5.sc_Morphology_Logistic_Regression/2.test_model/scripts/test_single_class_model.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import ast
+import itertools
+import pathlib
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pyarrow.parquet as pq
+import seaborn as sns
+import toml
+from joblib import dump, load
+from sklearn.exceptions import ConvergenceWarning
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import (
+    accuracy_score,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+)
+from sklearn.model_selection import GridSearchCV, StratifiedKFold, train_test_split
+from sklearn.utils import parallel_backend, shuffle
+
+# In[2]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+aggregation = False
+nomic = False
+flag = True
+control = "DMSO_0.100_DMSO_0.025"
+treatment = "Thapsigargin_1.000_DMSO_0.025"
+
+
+# In[3]:
+
+
+print(cell_type, aggregation, nomic, flag, control, treatment)
```

Comment:
Same comment here (Don't think this cell is necessary).

Reply:
addressed via deletion